src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -1077,6 +1085,12 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
       vm.stateManager.checkpoint();
 
       vm.on("step", (event: InterpreterStep) => {
+        const logs = maybeGetLogs(event);
+        if (logs) {
+          options.logging.logger.log(...logs);
```

Comment:
`logging.logger.log(...logs)` is in my top 10 favorite lines of code in ganache :laughing: